intelligence-services/latest/config/transcribe.md

```diff
@@ -0,0 +1,432 @@
+---
+title: Custom configuration - Transcribe
```

Comment:
Transcribe does not need a custom  configuration, you can remove this (that is showing the textract configuraiton in fact) Transcribe only needs the regular config you added here https://github.com/Alfresco/docs-alfresco/pull/192/files#diff-262981f6c5fee5870d1039c3db059a8be02435108d6eace92a7eee307fa2d3a9R243